src/main/java/chatbot/command/CommandOutput.java

```diff
@@ -0,0 +1,20 @@
+package chatbot.command;
+
+/**
+ * Output of commands.
+ */
+public class CommandOutput {
+    public final String output;
```

Comment:
As stated in the [coding standard](https://se-education.org/guides/conventions/java/intermediate.html#variables), the variables are declared as public since this is a _data class with no behaviour_.